model/utils.py

```diff
@@ -52,7 +52,18 @@ def collect_stats(train_loader,args):
             sum = sum_mel,
             sum_square = sum_square_mel)
     
-
+def get_MCD(true_spec, pred_spec,sr):
+    #y1, sr1 = sf.read(os.path.join(wav_path,str(i)+'.wav'))
+    #y2, sr2 = sf.read(os.path.join(wav_path,str(i)+'_true.wav'))
+    mfcc1 = librosa.feature.mfcc(y=pred_spec, sr=sr,n_mfcc=24)
```

Comment:
be careful about the calculation scale, refer to https://dsp.stackexchange.com/questions/56391/mel-cepstral-distortion